src/components/DiagramFrame/DiagramFrame.vue

```diff
@@ -158,7 +156,8 @@ export default {
       });
     },
     zoomIn() {
-      this.setScale(this.scale + 0.1);
+      const newScale = this.scale + 0.1 > 1 ? 1 : this.scale + 0.1;
```

Comment:
const newScale = Math.min(1, this.scale + 0.1)

Reply:
addressed.